src/main/java/io/github/amelonrind/stereopsis/Stereopsis.java

```diff
@@ -74,7 +74,7 @@ public static void resetHudOffset() {
     public static float getHudOffset() {
         if (hudOffset == -1.0f) {
             int width = mc.getWindow().getScaledWidth();
-            hudOffset = width / 4.0f + (Config.get().flipView ? xOffset : -xOffset) * width + 120 / screenAspectRatio;
+            hudOffset = (width / 4.0f + -xOffset * width + (Config.get().hudOffset) / screenAspectRatio);
```

Comment:
I really doubt that this'll look normal in AR view...
But I'll just believe you because I don't have AR glasses.
Just in case, I'll add that as an option.